src/core/api/Providers/SuppliersProvider.ts

```
@@ -0,0 +1,22 @@
+    static getSupplier(id: number) {
+        return api.get(`/suppliers/${id}`);
+    }
+
+    static createSupplier(payload: object) {
+        return api.post("/suppliers", payload);
+    }
+
+    static deleteSupplier(id: number) {
+        return api.delete(`/suppliers/${id}`);
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)